src/components/RedeemForm.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+import withIntl from '../lib/withIntl';
+
+import { Flex } from 'grid-styled';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { P } from './Text';
+import InputField from './InputField';
+import styled from 'styled-components';
+
+const Description = styled(P)`
```

Comment:
The `P` text component accepts `color`, `fontSize`, `lineHeight`, and `m` props, so it doesn't need to be extended like this.